monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -32,7 +32,7 @@
 from monai.utils import ensure_tuple
 
 logger = get_logger(module_name=__name__)
-ALGO_HASH = os.environ.get("MONAI_ALGO_HASH", "d7bf36c")
+ALGO_HASH = os.environ.get("MONAI_ALGO_HASH", "dc68e4d")
```

Comment:
this commit is on your branch https://github.com/mingxin-zheng/research-contributions/tree/fix_lr_stepsize but not on https://github.com/Project-MONAI/research-contributions/commits/main

Reply:
@wyli thank you for pointing it out. I updated the hash as well as the zip in the release page. Btw, let's keep both zipped files in the release page as the dc68e4d is used elsewhere.